CMakeLists.txt

```diff
@@ -269,7 +230,7 @@ endif()
 find_package(PkgConfig REQUIRED) 
 if (ASAGI)
   # todo warn if netcdf is off
-  pkg_check_modules(ASAGI REQUIRED asagi) # asagi_nompi?
+  pkg_check_modules(ASAGI REQUIRED IMPORTED_TARGET asagi) # asagi_nompi?
   target_compile_definitions(SeisSol-lib PUBLIC USE_ASAGI)
   target_link_libraries(SeisSol-lib PUBLIC ${ASAGI_STATIC_LDFLAGS})
   target_include_directories(SeisSol-lib PUBLIC ${ASAGI_INCLUDE_DIRS})
```

Comment:
Is there still code, which uses ASAGI directly in SeisSol? I think everything should be behind easi now?

Reply:
I think so too.

Reply:
AsagiModule.h for MPI init. Could be at some point moved to easi I guess